### DISCUSSION TITLE
Backup list should be paginated

### DISCUSSION THREAD

**Discussion by krizzdev** [spatie/nova-backup-tool#103] [Ideas]
Firstly thanks for the great package.
I like the way of simply accessing the latest backups, and download them or create manual ones.
I have setup laravel-backup for a project which generates hourly backups on s3.
Currently we have 200 backups on s3. 
Locally with only a few backups the nova backup tool works and the backups were displayed.
But when I set the config to s3, the nova backup tool don't load the view.
`Failed to load resource: the server responded with a status of 504`

Is there a way to only load and display the last 10 backups or reduce/filter the amount of displayed backups?

**Comment by freekmurze** [comment]
There currently is no way to do this, but I'd welcome a PR that adds pagination.

**Comment by krizzdev** [comment]
Thanks for your answer. Currently I have not enough free time to fix this. I am not that familiar with the internal flysystem yet. But maybe after Christmas ;)
